lib/schemas/simpleJSONataFilter.in.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
If we are going with the JSONata expression in the config, I think you can remove the entire metadata for this action.